stock_picking_delivery_info_computation/views/delivery_view.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='number_of_packages']" position="after">
```

Comment:
We should add this field as it has been removed and it's important for the aim on the module: delivery info.